<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CTAnalyzerX</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="centralLayout">
    <item>
     <layout class="QVBoxLayout" name="panelControlsLayout">
      <item>	  
       <widget class="VolumeControlsWidget" name="volumeControlsWidget"/>	   
      </item>
      <item>
       <widget class="QGroupBox" name="groupAnalysis">
        <property name="title">
         <string>Analysis</string>
        </property>
        <layout class="QVBoxLayout">
         <item>
          <widget class="QPushButton" name="btnRunAnalysis">
           <property name="text">
            <string>Run Analysis</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="txtAnalysisOutput"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="LightBoxWidget" name="lightBoxWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>600</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionScreenshot"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="text">
    <string>Take Screenshot</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
	<class>VolumeControlsWidget</class>
		<extends>QWidget</extends>
		<header>VolumeControlsWidget.h</header>
</customwidget>
  <customwidget>
   <class>LightBoxWidget</class>
   <extends>QLabel</extends>
   <header>LightBoxWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
